openhtf/plugs/usb/filesync_service.py

```diff
@@ -223,7 +223,7 @@ def _check_for_fail_message(self, transport, exc_info, timeout):  # pylint: disa
       if sys.exc_info()[0] is usb_exceptions.AdbRemoteError:
         raise
     # Otherwise reraise the original exception.
-    raise exc_info[0], exc_info[1], exc_info[2]
+    raise exc_info[0](exc_info[1]).with_traceback(exc_info[2])
```

Comment:
I think this isn't Python2 compatible:
http://python-future.org/compatible_idioms.html#raising-exceptions

I think we might need `raise_with_traceback` here instead.